keyrings/alt/file.py

```diff
@@ -1,34 +1,36 @@
 from __future__ import with_statement
 
 import os
-import getpass
-import base64
 import sys
 import json
+import getpass
 
 from keyring.py27compat import configparser
 
 from keyring.util import properties
 from keyring.util.escape import escape as escape_for_ini
 
-from . import file_base
-
+from keyrings.alt.file_base import (
```

Comment:
If it turns out that `file_base.decodebytes` is too wordy below, then maybe add an additional import for those two functions:

```
from .filebase import decodebytes, encodebytes
```